tests/nightly/test_large_array.py

```diff
@@ -455,6 +455,21 @@ def npy_instance_norm(data, gamma, beta, axis, eps=1E-5):
         assert_almost_equal(out, out_nd.asnumpy(), forward_check_eps,
                             forward_check_eps)
 
+    def check_rnn():
+        data = nd.random_normal(shape=(2**28, 4, 4))
+        parameters = nd.random_normal(shape=(7,))
+        state = nd.random_normal(shape=(1, 4, 1))
+        mode = 'rnn_relu'
+        state_size = 1
+        num_layers = 1
+
+        out = nd.RNN(data=data, parameters=parameters, state=state, mode=mode,
+                     state_size=state_size, num_layers=num_layers)
+
+        assert out.shape[0] == 268435456
```

Comment:
Please use constants for constant values. Its good practise and you may need to re-use them in future tests too.